l20n/migrate/__init__.py

```diff
@@ -0,0 +1,5 @@
+# coding=utf8
+
+from context import MergeContext
+from transforms import CONCAT, COPY, EXTERNAL, PLURALS, REPLACE, SOURCE
+from changesets import convert_blame_to_changesets
```

Comment:
I think we only need a release-cycle of compat for those migrations, and can remove those we applied on mergeday from the repo.
But yes, that means that for existing migrations, we'd need to port the migrations. Not sure if that's a bug or a feature. If we fix stuff, not having to investigate the impact on old migrations might hide bugs in new code or old migrations. If we change stuff, it might not be all that hard to port the old migrations?